### DISCUSSION TITLE
Archiving direct chats

### DISCUSSION THREAD

**Discussion by madduck** [element-hq/element-meta#2189] [Ideas]
I'm an early adopter and power-user of Matrix. However, I found that I am hesitant opening direct chats, and I was trying to get to the bottom of that.

One of the reasons is that I already have too many rooms, to the point that it's hard for me to find any particular room at any time. So when I am about to ask a colleague a once-off question, part of my brain goes "I won't make another room just for that".

I'm aware I can tag rooms low-prio, but that's a permanent flag.

I'd like to see a temporary flag "archived", similar to what Whatsapp does: when I am done asking my colleague, I can archive the chat, and it just disappears from my view. It still shows when I click on the colleagues profile, and should the colleague write a message, then it gets unarchived right away, but otherwise the room just isn't shown anymore.

Thanks for your consideration!

**Comment by disambiguator** [comment]
+1. I think this is a great idea and a really common workflow for those of us that have "Inbox Zero" type mentalities when it comes to messaging.

**Comment by HammyHavoc** [comment]
After bridging all my different messaging apps into Matrix and thus Element, it's really quite messy scrolling through hundreds of no longer relevant chats. Would make organizing myself much easier if archival ala WhatsApp was a thing, especially if when a new message in an archived chat happened, it ended up being removed from `Archived` status and being brought to my attention. As it stands, it's a total mess.

**Comment by daniellekirkwood** [comment]
Thanks for the suggestion! I'm going to go ahead and move this issue to our Discussions section as it's not something we're working on right now but want to ensure that this feedback is captured and searchable in the future

**Comment by il0** [comment]
I fully agree with these! Without folders or at least the most basic archive functionality, Element X / Matrix itself is pretty useless for a person who has hundreds of contacts.

**Comment by HammyHavoc** [comment]
Just a fleeting thought—would be interesting if archived chats that are via bridges don't check as frequently for new content where appropriate when configured to do so. Obviously comes down to how the bridge works, but if I have a few thousand chats on some bridges, there's no way that the overwhelming majority of them are going to be relevant very often.

Scale that across the other bridge users on the same server, it's got to be at least slightly meaningful in terms of lesser resource consumption—not saying it's heavy, but I think that over time, these things can probably become needlessly cumbersome than they otherwise could be.

Likewise, when signing into a Matrix client on a device, I don't need all of these chats being synced, so an `archived` status makes a lot of sense to avoid pointless activity.

**Reply by beedell-roke** [reply]
@HammyHavoc, I agree. The vast majority of my low-priority chats I must remain a member of, but shall likely never be utilised again. I don't need to synchronise them.

**Reply by HammyHavoc** [reply]
> @HammyHavoc, I agree. The vast majority of my low-priority chats I must remain a member of, but shall likely never be utilised again. I don't need to synchronise them.

Where I do also see a lot of untapped value is in using server-side search amidst unsynced `archived` messages to surface whichever piece of information or media item that's relevant to the search query. It makes more sense than syncing all the messages only to then search through them locally. Could really reduce needless data transfer from the server to clients.

**Reply by beedell-roke** [reply]
@HammyHavoc, this seems like something worth suggesting separately. I'll support it.

**Reply by HammyHavoc** [reply]
Good thinking! Here's the discussion for the idea: https://github.com/element-hq/element-meta/discussions/2929
